cluster/examples/kubernetes/ceph/psp.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: rook
+spec:
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+    - 'configMap'
+    - 'emptyDir'
+    - 'projected'
+    - 'secret'
+    - 'downwardAPI'
+    - 'hostPath'
+    - 'flexVolume'
+  hostPID: true
+  hostNetwork: true
+  hostIPC: true
+  hostPorts:
```

Comment:
Do you think it would be good to add a comment that `hostPorts` are only needed when running with `hostNetwork: true` in the CephCluster?

Reply:
Sure, I'll do that! Thanks!